mix.exs

```diff
@@ -68,7 +52,7 @@ defmodule Crawly.Mixfile do
       {:gollum, "~> 0.4.0", hex: :new_gollum},
       {:plug_cowboy, "~> 2.0"},
       {:credo, "~> 1.5.0", only: [:dev, :test], runtime: false},
-      {:ex_doc, "~> 0.19", only: :dev, runtime: false},
+      {:ex_doc, ">= 0.0.0", only: :dev, runtime: false},
```

Comment:
any reason why this change is necessary?
It would be preferable to leave as is to prevent major ver api changes from breaking docs generated.